docs/data/material/customization/theming/theming.md

```diff
@@ -114,6 +114,12 @@ You can extend the outer theme by providing a function:
 
 {{"demo": "ThemeNestingExtend.js"}}
 
+## CSS variables provider
+
+The `CssVarsProvider` is built on top of the `ThemeProvider` with extra features like theme CSS variables generation, built-in color scheme synchronization, user's system preference, and more.
```

Comment:
```suggestion
The `CssVarsProvider` is built on top of the `ThemeProvider` with extra features like theme CSS variables generation, built-in color scheme synchronization with the user's system preference, and more.
```
"user's system preference" by itself was vague; I know it means _color mode_ system preference, but that was implicit. Maybe connecting it like so makes it clearer?